### DISCUSSION TITLE
Java on MAc

### DISCUSSION THREAD

**Discussion by stanleyloftness** [edwardkort/WWIDesigner#100] [Setup] [answered]
Edward,

Apple no longer supports the version of Java that you have been using.  Is it possible for you to port your program to a newer version of Java.

Stan

**Comment by edwardkort** [comment, marked as answer]
Hi Stan,

The current version of WIDesigner runs on all java versions through 12. Please look at the java readme file with the distribution, because there are a few library-based hoops that you need to jump through for these newer versions. Hopefully JideSoft will resolve these issues in the foreseeable future.

You should also be aware that Oracle has changed its licensing agreement in java versions 9 and beyond. And, as far as I know, version 8 is the last version with an automatic updating system.

Politics in the software world.

Cheers,
Edward
